Research/ConfluenceAndCausalInvariance/ConfluenceAndCausalInvariance.md

```diff
@@ -0,0 +1,151 @@
+# Confluence and Causal Invariance
+
+## Introduction
+
+There are claims made in the [Wolfram Physics Project](https://www.wolframphysics.org) about the equivalence of
+confluence and causal invariance.
+For example, consider
+[the glossary](http://web.archive.org/web/20201010194129/https://www.wolframphysics.org/glossary/#CausalInvariance) on
+the [Wolfram Physics website](https://www.wolframphysics.org):
+
+> Causal Invariance: A property of multiway graphs whereby all possible paths yield the isomorphic causal graphs.
+> When causal invariance exists, every branch in the multiway system must eventually merge.
+> Causal invariance is a core property associated with relativistic invariance, quantum objectivity, etc.
+> In the theory of term rewriting, a closely related property is confluence.
+> In a terminating system, causal invariance implies that whatever path is taken, the "answer" will always be the same.
+
+> Confluence: A simplified form of causal invariance considered in term rewriting systems such as ones that reach fixed
+> points.
+
+In turn, the standard definition of
```

Comment:
I was a little confused here because I expected you to quote the glossary's definition of confluence, but it doesn't have one. Maybe it's wroth adding a sentence to say "The glossary does not define confluence, but we can assume it has the standard meaning and the one given in Gorard's paper".